packages/utils/src/formatDataValueByType.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Tupaia Web
+ * Copyright (c) 2019 - 2021 Beyond Essential Systems Pty Ltd.
+ * This source code is licensed under the AGPL-3.0 license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
```

Comment:
Since it's in @tupaia/utils, perhaps we can change it to:

```
/**
 * Tupaia
 * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
 */
```

Reply:
Thanks Chris for focusing on details. 👍 I just copy and paste and didn't look at it closely. I will change it later.